local.go

```diff
@@ -164,6 +173,12 @@ func (l *LocalTest) GenRosterFromHost(servers ...*Server) *Roster {
 	return list
 }
 
+func (l *LocalTest) panicClosed() {
```

Comment:
I don't really like this because whoever is writing a new function will need to remember to call `panicClosed`. However, I can't think of any other way to do it and it's not a huge problem if it's not called, it'll just take longer to debug.